src/features/cards/organisms/card-item.js

```diff
@@ -76,3 +86,48 @@ const CardBox = styled(Card)`
   max-height: 24rem;
   overflow-y: hidden;
 `
+
+const PopUpButton = () => {
+  const [popup, setPopup] = useState(false)
```

Comment:
Maybe better name is `[opened, setOpened]`

Reply:
Not entirely agree. Opened is too common, setPopup is making it close too. I would like to keep as it is it for now.

Reply:
solved